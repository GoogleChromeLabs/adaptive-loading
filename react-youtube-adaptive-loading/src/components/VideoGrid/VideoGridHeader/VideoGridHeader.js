import React from 'react';
import './VideoGridHeader.scss';

export function VideoGridHeader(props) {
  return (
    <div className='video-grid-header'>
      <span className='title'>{props.title}</span>
    </div>
  );
}
