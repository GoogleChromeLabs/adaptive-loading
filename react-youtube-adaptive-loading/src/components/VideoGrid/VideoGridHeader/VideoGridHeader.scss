.video-grid-header {
    padding-bottom: 24px;
    padding-top: 14px;
    .title {
      font-size: 1.2rem;
      font-weight: bold;
    }
  }