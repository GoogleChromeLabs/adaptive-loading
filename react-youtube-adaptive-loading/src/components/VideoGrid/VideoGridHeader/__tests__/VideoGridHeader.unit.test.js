import React from 'react';
import {shallow} from 'enzyme';
import {VideoGridHeader} from '../VideoGridHeader';

describe('VideoGridHeader', () => {
  test('renders without props', () => {
    const wrapper = shallow(<VideoGridHeader/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders with empty string header', () => {
    const wrapper = shallow(<VideoGridHeader title=''/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders with title', () => {
    const wrapper = shallow(<VideoGridHeader title='Autos & Vehicles'/>);
    expect(wrapper).toMatchSnapshot();
  });
});