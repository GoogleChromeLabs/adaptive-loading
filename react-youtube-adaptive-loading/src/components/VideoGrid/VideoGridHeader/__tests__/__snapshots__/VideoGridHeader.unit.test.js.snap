// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoGridHeader renders with empty string header 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;

exports[`VideoGridHeader renders with title 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  >
    Autos & Vehicles
  </span>
</div>
`;

exports[`VideoGridHeader renders without props 1`] = `
<div
  className="video-grid-header"
>
  <span
    className="title"
  />
</div>
`;
